test/test_tensorexpr.py

```diff
@@ -376,6 +376,65 @@ def easy(x, y):
         x = traced(a, b)
         np.testing.assert_allclose(a.numpy() + b.numpy(), x.numpy())
 
+    def test_double(self):
+        def easy(x, y):
+            aaa = torch.add(x, y)
+            return aaa
+
+        traced = torch.jit.trace(
+            easy,
+            (torch.rand(1024, dtype=torch.float64), torch.full((1024,), 0.5, dtype=torch.float64)),
+        )
+
+        a = torch.rand(1024, dtype=torch.double)
+        b = torch.full((1024,), 0.5, dtype=torch.double)
+        x = traced(a, b)
+        np.testing.assert_allclose(a.numpy() + b.numpy(), x.numpy())
+
+    def test_short(self):
+        def easy(x, y):
+            aaa = torch.add(x, y)
```

Comment:
I'm not sure the fuser gets invoked if there's just one `aten` op in the region.  You might need to do something like `x * y * y`.